### PR TITLE
Updates to qube-config.md

### PR DESCRIPTION
**Pull request by SarahJohnsonONS**

### CODE REVIEW

File: external-docs/docs/guides/configuration/qube-config.md [hunk 1]
```diff
@@ -121,56 +125,60 @@ There are several configuration options available across column types except obs
...
 | `type`              | The type of the column (Required)                                                                                                                                                                                    | *dimension*                                                                      |
+| `from_template`     | Use a [column template](templates.md)                                                                                                                                                                                | *none*                                                                           |
 | `label`             | The title of the column (Optional)                                                                                                                                                                                   | The capital case of the header in the csv file with spaces replacing underscores |
```
Comment (NickPapONS) [inline external-docs/docs/guides/configuration/qube-config.md:136]: I agree with what you mentioned in your notes about making the casing consistent for acronyms like CSV or URI. This is a place where an inconsistency like that exists, I found that it is written as "csv" within table text, but outside of the tables it is always written as "CSV". A very small (maybe meaningless) inconsistency but maybe still worth looking at.

File: external-docs/docs/guides/configuration/qube-config.md [hunk 2]
```diff
@@ -121,56 +125,60 @@ There are several configuration options available across column types except obs
...
 | `description`       | A description of the contents of the column (Optional)                                                                                                                                                               | *none*                                                                           |
 | `from_existing`     | The uri of the resource for reuse/extension (Optional)                                                                                                                                                               | *none*                                                                           |
```
Comment (NickPapONS) [inline external-docs/docs/guides/configuration/qube-config.md:138]: Similar to the csv/CSV notes, the term uri always appears in lower case in this documentation page, however in other parts of the documentation (e.g. the "Working with URIs" section) it is written in caps. I'm not sure which is best, as lower case is obviously suited for file extensions, but maybe caps is better for just normal references?

File: external-docs/docs/guides/configuration/qube-config.md [hunk 3]
```diff
@@ -46,33 +50,33 @@ A column is assumed to be a dimension unless otherwise configured using the `typ
...
 
-The *unit* component is a type of attribute column which provides the units of the observation.
+The *unit* column is a type of attribute column which provides the units of the observation.
 
 ### Observations
 
-The *observation* column are the numeric values of the observation being recorded in the data set.
+The *observation* column contains the numeric values of the observations recorded in the data set.
 
 ### Using templates
```
Comment (NickPapONS) [inline external-docs/docs/guides/configuration/qube-config.md:82]: I followed up on your notes about the template page this links to having a broken link for the available templates(https://github.com/GSS-Cogs/csvcubed/tree/main/csvcubed/csvcubed/readers/cubeconfig/v1_0/templates) and you are right, it is incorrect. I believe the correct link is https://github.com/GSS-Cogs/csvcubed/tree/main/src/csvcubed/readers/cubeconfig/v1_0/templates
I know it is not part of the qube config documentation but well done for noticing that anyway

File: external-docs/docs/guides/configuration/qube-config.md [hunk 4]
```diff
@@ -8,23 +8,27 @@ This page discusses how to configure your cube with the greatest control and fle
 The `qube-config.json` file has two sections:
```
Comment (NickPapONS) [inline external-docs/docs/guides/configuration/qube-config.md:8]: This is just a personal thing that maybe only applies to me, but as someone looking at this for the first time I thought I might find it useful to either mention or link to the schema/its location more directly, do you think that might belong here or in the "writing a qube-config.json" guide that will be created in the future?

**Review by NickPapONS — COMMENTED** [review]
Nice changes, looks good! Made a lot of things easier to read and I think everything that was listed in the ticket has been corrected. I wrote a couple of comments about things from my notes that I would like to know your opinion on.

File: external-docs/docs/guides/configuration/qube-config.md (hunk 3 above)
Comment (SarahJohnsonONS) [reply external-docs/docs/guides/configuration/qube-config.md:82]: Thanks, amended the link

File: external-docs/docs/guides/configuration/qube-config.md (hunk 4 above)
Comment (SarahJohnsonONS) [reply external-docs/docs/guides/configuration/qube-config.md:8]: Thanks, have added this as a suggestion in the new ticket (#706)

File: external-docs/docs/guides/configuration/qube-config.md (hunk 1 above)
Comment (SarahJohnsonONS) [reply external-docs/docs/guides/configuration/qube-config.md:136]: Added to new ticket (#706)

File: external-docs/docs/guides/configuration/qube-config.md (hunk 2 above)
Comment (SarahJohnsonONS) [reply external-docs/docs/guides/configuration/qube-config.md:138]: Added to new ticket (#706)

File: external-docs/docs/guides/configuration/qube-config.md [hunk 5]
```diff
@@ -121,56 +125,60 @@ There are several configuration options available across column types except obs
...
+| `data_type`              | (Attribute Literals only) The [xml data type](https://www.w3.org/TR/xmlschema-2/#built-in-datatypes) of the contents of the column, if this is provided it becomes a Literal Attribute column (Optional)                                                                                                                                                                       | *none*                                                                           |
+| `values`                 | (New Resource Attributes only) If automatically-generated attributes are desired, a boolean value of `true` is used to signify to csvcubed to create attribute resources from values in this column; otherwise this should be a dictionary defining the attributes used in the column. See [Attribute values configuration](#attribute-values-configuration) for more details. | *none*                                                                           |
```
Comment (robons) [inline external-docs/docs/guides/configuration/qube-config.md:157]: Like the new section and the link!

**Review by robons — APPROVED** [review]

File: external-docs/docs/guides/configuration/units.md [hunk 6]
```diff
@@ -79,7 +79,7 @@ In the following example, we'll show how to define new units for `Pounds Sterlin
 }
 ```
 
-**N.B. you must ensure that your units column definition contains defintions for *all* of the units contained within your dataset.**
+**N.B. you must ensure that your units column definition contains definitions for *all* of the units contained within your dataset.**
```
[the remark this answers is not in the document]
Comment (SarahJohnsonONS) [reply external-docs/docs/guides/configuration/units.md:82]: Yes please!

[the remark this answers is not in the document]
Comment (canwaf) [reply external-docs/docs/guides/configuration/units.md:82]: Be careful of adding in spell check, @robons, you're going to have to manage a lot of exceptions. Qube, RDF, CSVW, SKOS, XKOS... I don't know if this is going to *save time*.

**Pull request merged**